react-components/src/architecture/concrete/axis/AxisThreeView.ts

```diff
@@ -607,6 +601,48 @@ function getTickDirection(faceIndex1: number, faceIndex2: number, target: Vector
   return target;
 }
 
+function incrementToString(value: number): string {
```

Comment:
Could we instead use this function? https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_objects/Number/toPrecision

Reply:
Jeg slet med den funksjonen. Nei, vi kan ikke bruke slike fuksjoner. Vi vet vi ikke noe om presisjonen eller mangnituden av value. Kan vøre hva som helst egentlig.